nntrainer/layers/multi_head_attention_layer.cpp

```diff
@@ -265,22 +280,23 @@ void MultiHeadAttentionLayer::finalize(InitLayerContext &context) {
   }
   /** tensor for attention weight */
   TensorDim attention_weight_dim(
-    {batch_size, num_heads, query_height, key_height});
+    {batch_size, num_heads, query_height, key_height}, tensor_type);
   weight_idx[AttentionParams::attention_weight] = context.requestTensor(
     attention_weight_dim, "attention_weight", Tensor::Initializer::NONE, true,
     TensorLifespan::ITERATION_LIFESPAN);
   if (dropout_rate > epsilon) {
     /** tensor for dropout mask */
     TensorDim dropout_mask_dim(
-      {batch_size, num_heads, query_height, key_height});
+      {batch_size, num_heads, query_height, key_height}, tensor_type);
     weight_idx[AttentionParams::dropout_mask] = context.requestTensor(
       dropout_mask_dim, "dropout_mask", Tensor::Initializer::NONE, false,
       TensorLifespan::ITERATION_LIFESPAN);
   }
 
   /** tensor for attention output */
   TensorDim attention_output_dim(
-    {batch_size, 1, query_height, num_heads * projected_value_dim_prop});
+    {batch_size, 1, query_height, num_heads * projected_value_dim_prop},
+    tensor_type);
```

Comment:
I wonder if this is the tensor_type or {context.getFormat(), context.getActivationDataType()}. please check once again.

Reply:
Just fixed it. Also fixed setActivationFunc for the same reason.